website/static/js/ads.js

```diff
@@ -0,0 +1 @@
+var canRunAds = true;
```

Comment:
Is there some documentation about why this works? If so, add a comment with a link to that documentation (a stack overflow question or ad-blocker FAQ or whatever) so if it stops working in the future we'll have an idea where to check.